modules/map/hdmap/adapter/xml_parser/error_codes.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2017 The Apollo Authors. All Rights Reserved.
```

Comment:
There's a common/error_code proto. Remove this file and put all the error codes there.